analyzer/src/funTest/kotlin/DotNetTest.kt

```diff
@@ -68,8 +66,7 @@ class DotNetTest : StringSpec() {
         }
 
         "Definition File is correctly mapped" {
-            val mapper = XmlMapper().registerKotlinModule()
-            val result = mapper.readValue<List<DotNet.Companion.ItemGroup>>(packageFile)
+            val result = DotNetSupport.mapper.readValue<List<DotNet.Companion.ItemGroup>>(packageFile)
```

Comment:
I believe this rather generic test was added by @neubs-bsi because there were general issues with deserializing XML with multiple nodes of the same name using Jackson, see https://github.com/FasterXML/jackson-dataformat-xml/issues/205. I'd like to keep if for now as I'll be doing more XML-related stuff soon, but I agree it could be deleted eventually later if we feel confident that the general XML-deserialization issue has been solved for good.